src/block_disagg/block_disagg_write.c

```diff
@@ -116,17 +128,50 @@ __wt_block_disagg_write_internal(WT_SESSION_IMPL *session, WT_BLOCK_DISAGG *bloc
      */
     blk->flags = 0;
     if (data_checksum)
-        F_SET(blk, WT_BLOCK_DATA_CKSUM);
+        F_SET(blk, WT_BLOCK_DISAGG_DATA_CKSUM);
+
+    /*
+     * XXX temporary measure until we the block header at the beginning of the data. Set the block
```

Comment:
There seems something committed in the comment. until we "do" what to the block header?

Reply:
I've added more to the comment, it's just that we have two sets of encrypt/compress flags, we'll be able to get rid of this situation once we put the block header first.